### PR TITLE
Fixed bug with missing byte in u128 scale encoded TCP packet.

### PR DESCRIPTION
**Pull request by humb1t**
Signed-off-by: Nikita Puzankov <humb1t@yandex.ru>

### CODE REVIEW

File: iroha_network/src/lib.rs [hunk 1]
```diff
@@ -18,7 +18,7 @@ pub mod prelude {
     pub use crate::{AsyncStream, Network, Request, Response, State};
 }
 
-pub const BUFFER_SIZE: usize = 2048;
+pub const BUFFER_SIZE: usize = 1024;
```
Comment (eadventurous) [inline iroha_network/src/lib.rs:21]: What is the purpose of this change? I am not against, just asking :)

File: iroha_network/src/lib.rs [hunk 2]
```diff
@@ -62,7 +62,7 @@ impl Network {
             .await
             .map_err(|e| e.to_string())?;
         stream.flush().await.map_err(|e| e.to_string())?;
-        let mut buffer = [0u8; BUFFER_SIZE];
+        let mut buffer = vec![0u8; BUFFER_SIZE];
```
Comment (eadventurous) [inline iroha_network/src/lib.rs:65]: Also here why change to `vec`?

File: iroha_network/src/lib.rs (hunk 1 above)
Comment (humb1t) [reply iroha_network/src/lib.rs:21]: Want make our constraints stricter.

File: iroha_network/src/lib.rs (hunk 2 above)
Comment (humb1t) [reply iroha_network/src/lib.rs:65]: Was made in alignment with API docs. We need to check that that approach will work even if size of transaction exceeds initial buffer size.
